src/components/Input.jsx

```diff
@@ -0,0 +1,11 @@
+export default function Input({ id, type = "text", placeholder, ...props }) {
+  return (
+    <input
+      id={id}
+      type={type}
+      placeholder={placeholder}
+      className="w-full p-4 bg-[#F3F4F6] placeholder:text-[#9CA3AF] rounded-md"
+      {...props}
+    />
+  );
+}

```

Comment:
💊 제안
깔끔한 분리 좋습니다! 다만 추후 classname을 받을 수 있게 하시면 더 좋을 것 같아요.
이렇게 classname을 외부에서 받아야 기본 인풋위에 필요한 스타일을 적용시키며 확장성 있게 사용할 수 있습니다.

```suggestion
export default function Input({ id, type = "text", placeholder, classname ="", ...props }) {
  return (
    <input
      id={id}
      type={type}
      placeholder={placeholder}
      className={`w-full p-4 bg-[#F3F4F6] placeholder:text-[#9CA3AF] rounded-md ${classname}`}
      {...props}
    />
  );
}

```

참고로 tailwind의 경우 작성한 클래스 네임의 순서가 뒤에 있다고 적용이 보장되지 않기 때문에 
아래 문서를 읽어보시고 이러한 특징도 고려해서 컴포넌트를 수정해보시면 더 좋겠습니다!

https://yoonho-devlog.tistory.com/199